arm-machinelearning/2016-05-01-preview/swagger/commitmentPlans.json

```diff
@@ -0,0 +1,903 @@
+                        "in": "path",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "$ref": "#/parameters/APIVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/CommitmentAssociation"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearning/commitmentPlans/{planName}/commitmentAssociations": {
+            "get": {
+                "tags": [
+                    "CommitmentAssociation"
+                ],
+                "operationId": "CommitmentAssociation_GetCommitmentAssociations",
+                "description": "Get all commitment associations for a parent commitment plan.",
+                "parameters": [
+                    {
+                      "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "name": "resourceGroupName",
+                        "description": "The resource group the resources are under.",
+                        "in": "path",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "planName",
+                        "description": "The name of the parent commitment plan.",
+                        "in": "path",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "$skipToken",
+                        "description": "Continuation token for pagination.",
+                        "in": "query",
+                        "required": false,
+                        "type": "string"
+                    },
+                    {
+                        "$ref": "#/parameters/APIVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/PaginatedResult[CommitmentAssociation]"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearning/commitmentPlans/{planName}/commitmentAssociations/{associationName}/move": {
+            "put": {
+                "tags": [
+                    "CommitmentAssociation"
+                ],
+                "operationId": "CommitmentAssociation_MoveCommitmentAssociationPut",
```

Comment:
How is the MoveCommitmentAssociationPut different from MoveCommitmentAssociationPost.
Both of them have same path parameters, body parameter and the same response type for 200. Do we really need both of them? If they are really required then how is the customer supposed to know when to use what? It is very confusing.